package/fedora/freecad.spec

```diff
@@ -197,13 +193,14 @@ LDFLAGS='-Wl,--as-needed -Wl,--no-undefined'; export LDFLAGS
        -DCMAKE_INSTALL_INCLUDEDIR=%{_includedir} \
        -DRESOURCEDIR=%{_datadir}/%{name} \
        -DFREECAD_USE_EXTERNAL_PIVY=TRUE \
-       -DFREECAD_USE_PCL=TRUE \
-       -DBUILD_QT5=ON \
-       -DSHIBOKEN_INCLUDE_DIR=%{_includedir}/shiboken2 \
-       -DSHIBOKEN_LIBRARY=-lshiboken2.%{py_suffix} \
+       -DFREECAD_USE_EXTERNAL_FMT=TRUE \
+       -DFREECAD_USE_PCL:BOOL=OFF \
```

Comment:
is there an issue with pcl in fedora?

Reply:
if this is to be set to off pcl-devel should be removed from dependencies I think

Reply:
Yes, pkg-config unfortunately fails if it is enable, but the dependency is there. And I have not had any success yet in resolving it.

Reply:
I will continue to try finding a solution. But keep FREECAD_USE_PCL explicitly off until a solution is found.